core/utils/object.js

```diff
@@ -19,15 +19,25 @@ goog.provide('Blockly.utils.object');
 
 /**
  * Inherit the prototype methods from one constructor into another.
- *
  * @param {!Function} childCtor Child class.
  * @param {!Function} parentCtor Parent class.
  * @suppress {strictMissingProperties} superClass_ is not defined on Function.
  */
 Blockly.utils.object.inherits = function(childCtor, parentCtor) {
+  // Set a .superClass_ property so that methods can call parent methods
+  // without hard-coding the parent class name.
+  // Could be replaced by ES6's super().
   childCtor.superClass_ = parentCtor.prototype;
+
+  // Link the child class to the parent class so that static methods inherit.
+  Object.setPrototypeOf(childCtor, parentCtor);
```

Comment:
Observation: `.setPrototypeOf` was not formally introduced until ES6, but AIUI you have confirmed that it is available on all our target platforms.

Reply:
Yes, it was introduced in IE11.